content/docs/2.7/operate/cluster.md

```diff
@@ -65,6 +65,17 @@ $env:KEDA_HTTP_DEFAULT_TIMEOUT=1000
 
 All applicable scalers will use this timeout. Setting a per-scaler timeout is currently unsupported.
 
+## HTTP Proxies
+
+Some scalers issue HTTP requests to external servers (i.e. cloud services). As certain companies require external servers to be accessed by proxy servers, adding the relevant environment variables to your deployment (HTTP_PROXY, HTTPS_PROXY, NO_PROXY, etc.) would allow the scaler to connect via the desired proxy.
```

Comment:
Looking good, just a minor nit, could you please replace the section `to your deployment` to `KEDA Operator and KEDA Metrics Server deployments` or something similar? It might be obvious, but just to be sure, that users need to add these to both deployments.